rabbit_mq/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "region" {
+  description = "AWS Default Region"
+  default     = "us-east-1"
+}
+
+variable "profile" {
+  type        = string
+  default     = "default"
+  description = "Credentials profile for AWS provider"
+}
+
+variable "user_tags" {
+  description = "Mandatory tags for the RabbitMQ resources"
+  type        = map(string)
+}
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type        = map(string)
+}
+
+variable "broker_name" {
+  type        = string
+  description = "Name of the Amazon MQ Broker"
+}
+
+variable "engine_type" {
+  default     = "RabbitMQ"
+  description = "This terraform module will currently support only RabbitMQ"
+}
+
+variable "engine_version" {
+  type        = string
+  default     = "3.8.11"
+  description = "This is the engine version for RabbitMQ"
+  validation {
```

Comment:
Is this required? Won't AWS handle the validation?

Reply:
removed